khal/khalendar/event.py

```diff
@@ -451,6 +458,8 @@ def relative_to(self, day, full=False):
                 body += ', ' + self.description.strip()
             if self.location.strip() != '':
                 body += ', ' + self.location.strip()
+            if self.categories.strip() != '':
+                body += ', ' + self.categories.strip()
```

Comment:
I think it would make sense to group the calendar selector and the categories textfield together.